web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/config/GetStaffInfoFacadeService.java

```diff
@@ -29,4 +29,6 @@ public interface GetStaffInfoFacadeService {
     String getLoggedInStaffHsaId();
 
     boolean isLoggedInUserDoctor();
+
+    List<String> getUnits();
```

Comment:
Could this be be changed to a little more readable name. You don't understand what it returns until you read the actual implementation.